backend/apps/notifications/tasks.py

```
@@ -133,3 +133,74 @@ def send_bulk_email(payload):
+    from .models import NotificationPreference, Notification
+
+    prefs = NotificationPreference.objects.exclude(digest_frequency="none").select_related("user", "user__user_profile")
+    
+    for pref in prefs:
+        if not pref.digest_time:
+            continue
+            
+        user = pref.user
+        try:
+            tz = zoneinfo.ZoneInfo(user.user_profile.timezone)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)